examples/n-body/tests/index.js

```diff
@@ -62,7 +66,7 @@ console.log("Took " + (time[0] * 1e3 + time[1] / 1e6) + "ms");
 console.log("\nWARMED UP:\n");
 
 console.log("Performing " + steps + " steps (AssemblyScript WASM) ...");
-time = test(nbodyWASM, steps);
+time = test(nbodyWAsmWASM, steps);
```

Comment:
WAsmWASM? :)

Reply:
I refactored this already=)

Reply:
See: https://github.com/AssemblyScript/assemblyscript/pull/88/commits/540dea56c9f6a1319933582c1da2953ddb8b318f

Reply:
Heh, what about `nbodyAS` and `nbodyRS`? :)

Reply:
Refactored to `nbodyASWasm`, `nbodyRSWasm` and `nbodyAsmJS`. WDYT?

Reply:
Hardest thing in programming? Naming things! 😅 (so, you don't like nbodyAS, nbodyRS, nbodyJS?)

Reply:
I never imagine that naming is so hard stuff 😅

Reply:
We need some philosophical conventions for naming. Something like Occam's razor=)